roles/epel_rpm_url/tasks/main.yml

```diff
@@ -21,14 +21,14 @@
 
 - name: Set EPEL base url
   set_fact:
-    epel_base: "https://download.fedoraproject.org/pub/epel/7/x86_64"
+    epel_base: "https://download.fedoraproject.org/pub/epel/7/x86_64/Packages"
 
 - name: Get subdir url
   set_fact:
     epel_subdir: "{{ epel_base }}/{{ rpm_name[0] }}"
 
 - name: Scrape HTML for package name
-  shell: curl -LsSk {{ epel_subdir }}/ | sed -n 's/.*"\({{ rpm_name }}.*.rpm\)\".*/\1/p'
+  shell: curl -LsSk {{ epel_subdir | quote }}/ | sed -n 's|.*\"\({{ rpm_name | quote }}.*.rpm\)\".*|\1|p'
   register: epel_output
   retries: 5
   delay: 60
```

Comment:
I noticed in the CI failures, the curl gets a 404 and sed no-ops but still results in a 0 return code and the test fails further down the line.  Maybe we should split this into two tasks--one to curl the epel subdir (retry until we don't get a 404), then one to grab the rpm name from the output?

Reply:
Seems reasonable, I'll look into it.

Although, the constant `curl` failures when talking to the Fedora mirrors is both troublesome and annoying.

Reply:
I agree its pretty annoying.  For some reason when I curl repeatedly on my local system, it worked every time.